test_agent/java/src/main/java/org/eclipse/uprotocol/TestAgent.java

```diff
@@ -139,7 +151,27 @@ private static void writeDataToTMSocket(JSONObject responseDict, String action)
         }
     }
 
-    private static UStatus handleSendCommand(Map<String, Object> jsonData) {
+    private static Object handleInitializeTransportCommand(Map<String, Object> jsonData) {
+        UUri uri = (UUri) ProtoConverter.dictToProto((Map<String, Object>) jsonData.get("data"), UUri.newBuilder());
+        String testID = (String) jsonData.get("test_id");
+        try {
+            if (transportName.equals("socket")) {
+                transport = new SocketUTransport(uri);
+            } else {
+                UStatus status = UStatus.newBuilder().setCode(UCode.INVALID_ARGUMENT).setMessage("").build();
+                sendToTestManager(status, ActionCommands.INITIALIZE_TRANSPORT, testID);
+            }
+        } catch (IOException e) {
+            e.printStackTrace();
+            UStatus status = UStatus.newBuilder().setCode(UCode.INVALID_ARGUMENT).setMessage("").build();
+            sendToTestManager(status, ActionCommands.INITIALIZE_TRANSPORT, testID);
+        }
+        UStatus status = UStatus.newBuilder().setCode(UCode.OK).setMessage("").build();
+        sendToTestManager(status, ActionCommands.INITIALIZE_TRANSPORT, testID);
+        return null;
+    }
```

Comment:
UStatus code should match other testagent response 

C++ and Python are using FAILED_PRECONDITION

Reply:
fixed